paddle/fluid/framework/details/ssa_graph.h

```diff
@@ -25,12 +25,22 @@ namespace paddle {
 namespace framework {
 namespace details {
 
+// A SSA graph used by parallel executor.
```

Comment:
Maybe add a link to the Wikipedia page to explain what SSA is.

Reply:
It will be added in next PR.